src/main/resources/task-script-executor.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set +e
+
```

Comment:
I think we both agree this is not the simplest code ever made by mankind. please, drop some comments.

Reply:
Done.